src/app/donation-start/donation-start.component.ts

```diff
@@ -871,4 +875,12 @@ export class DonationStartComponent implements AfterContentChecked, OnDestroy, O
         );
     };
   }
+
+  private setchampionOptInValidity() {
+    if (this.campaign?.championOptInStatement !== null) {
```

Comment:
This conditional should be identical to the template so we can't have glitches with e.g. blank strings where only one of the two matches. May be simpler to set a public boolean var and use that in both?